apps/i18n/common/en_us.json

```diff
@@ -83,7 +83,9 @@
   "assignCourse": "Assign Course",
   "assignConfirm": "Are you sure you want to assign \"{assignmentName}\" to \"{sectionName}\"?",
   "assignHiddenUnitConfirm": "It looks like you previously hid \"{assignmentName}\" from section \"{sectionName}\". Do you still want to assign this unit and make it visible?",
+  "assigned": "Assigned",
```

Comment:
looks like this string already existed on line 81

Reply:
oh, hah; looks like it was actually added in https://github.com/code-dot-org/code-dot-org/pull/31071